lib/webpack-cli.js

```diff
@@ -162,11 +162,14 @@ class WebpackCLI extends GroupHelper {
 
     runHelp(args) {
         const HelpGroup = require('./groups/help');
-        const { names } = require('./utils/commands');
-        const command = names.filter(name => {
+        const commandNames = require('./utils/commands').names;
+        const flagNames = require('./utils/core-flags').names;
+        const allNames = [...commandNames, ...flagNames];
+        const subject = allNames.filter(name => {
             return args.includes(name);
         })[0];
```

Comment:
```js
        const subject = allNames.filter(name => args.includes(name))[0];
```
Maybe :thinking:

Reply:
A little bit difficult to understand at first glance ...